admin_manual/maintenance/manual_upgrade.rst

```diff
@@ -2,6 +2,10 @@
 Manual ownCloud Upgrade
 =======================
 
+.. warning:: When upgrading from oC 9.0 to 9.1 with existing Calendars or Adressbooks
```

Comment:
People tend to have a look at the docs for their current version (e.g. 9.0 when being on 9.0) so they would miss this part when its only added to 9.1/master